Wauquelin/graph.gv

```diff
@@ -0,0 +1,8 @@
+digraph {
+    Xprime [label="X'"]
```

Comment:
I think X and Xprime should be in gray (lost ms.), no ?

Reply:
But X is the ms of Jean II:  lost but not hypothetical, no? I put X' n grey.

Reply:
Is the ms. of Jean II still extant ? I thought it was a lost (i.e., hypothetical) ms.

Reply:
Ok, I do believe only A, B, D, C are extant, as well as V (a single leave detached from a manuscript alas in private hands).

Reply:
Lost is not hypothetical (to me). Then what would we do of mss that do exist but that we do not have (often the case for modern literature)

Reply:
https://www.arlima.net/il/jean_wauquelin.html
http://jonas.irht.cnrs.fr/oeuvre/5169

Reply:
You can't be sure it exists if you do not have it.

Reply:
Otherwise, the original (as there always was one, obviously) would not be hypothetical.